tool/jt.rb

```diff
@@ -884,61 +881,53 @@ def check_test_port
   end
 
   def test_integration(env={}, *args)
-    env_vars   = env
-    jruby_opts = []
+    env = env.dup
+    
+    classpath = []
 
     if ENV['GRAAL_JS_JAR']
-      jruby_opts << '-J-cp'
-      jruby_opts << Utilities.find_graal_js
+      classpath << Utilities.find_graal_js
     end
 
     if ENV['SL_JAR']
-      jruby_opts << '-J-cp'
-      jruby_opts << Utilities.find_sl
+      classpath << Utilities.find_sl
     end
 
-    env_vars["JRUBY_OPTS"] = jruby_opts.join(' ')
-
+    unless classpath.empty?
+      env['JAVA_OPTS'] = classpath.map { |cp| "-cp #{cp}"  }.join(' ')
```

Comment:
Actually Java accepts a single `-cp` (but the launchers accept multiple), so maybe it's simpler to just:
`"-cp #{classpath.join(':')}"`